CONTRIBUTING.md

```diff
@@ -188,15 +188,19 @@ As subject matter experts, *code owners* approve pull requests on the packages t
 
 Acceptance of contributions follows the consensus-seeking process.
 
-All pull requests must be approved by a *collaborator* before the pull request can be accepted.
+All pull requests must be approved by an assigned *collaborator* before the pull request can be accepted. A *collaborator* will be assigned as the designated reviewer of a pull request within [a reasonable period of time](#response-time) of the pull request creation. Other users may provide reviews, however the *collaborator* assigned to the pull request will have final say on approval.
 
-Before a pull request is accepted, time should be given to receive input from *collaborators* or *code owners* with the expertise to evaluate the changes. The amount of time can vary but at least 3 days during the typical working week and 5 days over weekends should be given to account for international time differences and work schedules.
+Before a pull request is accepted, [a reasonable period of time](#response-time) should be given to receive input from *collaborators* or *code owners* with the expertise to evaluate the changes.
 
 When a pull request : (a) has a significant impact on the project, (b) is inherently controversial, or (c) has not reached consensus with *collaborators*; add a "status:controversial" label to the pull request for the *steering committee* to review the pull request. Pull requests labeled with "status:controversial" are not approved until the *steering committee* reviews the issue and makes a decision.
 
 Additionally, *owners*, can temporarily enable [interaction limits](https://help.github.com/articles/limiting-interactions-with-your-repository/) to allow a "cool-down" period when hot topics become disruptive.
 
-Specific *collaborators* or *code owners*  can be added to a pull request by including their user alias.
+Specific *collaborators* or *code owners* can be added to a pull request by including their user alias in the "Reviewers" section.
```

Comment:
Is the expectation that users add folks? Owners are added automatically, but if we are hoping to have collaborator assignment be what manages this we likely need to automate that on our end to ensure that we distribute the work equally and to each responsible party

Reply:
I'm changing this to assignee adding reviewers if they feel it's necessary.